examples/sglang_bench/bench_sharegpt.py

```
@@ -38,6 +38,234 @@
 
 _CONFIG_FILE = Path(__file__).parent / "model_configs.json"
 
+# Compute nvidia CUDA header paths once at startup.
+# pip-installed nvidia packages (nvidia-cuda-nvrtc-cu13 etc.) place nvrtc.h,
+# cuda_runtime.h etc. under site-packages/nvidia/cuXX/include.  These are not
+# on the default include path of the system compiler, but JIT kernels such as
+# flashinfer's TRT-LLM FMHA module need them.  Exporting CPATH makes them
+# visible to every subprocess (SGLang server, SSH peer worker) without patching
+# individual compile commands.
+_NVIDIA_INCLUDES: List[str] = sorted(
+    str(p) for p in Path(sys.prefix).glob("lib/python*/site-packages/nvidia/cu*/include")
```

Ruff (0.15.9)

[error] Undefined name `sys`

(F821)